server.js

```diff
@@ -5,17 +5,51 @@
  */
 
 const express = require('express')
+const request = require('request')
+const Config = require('./src/Config')
 const versoSpoof = require('./src/versoSpoof')
 
 const port = 8000
 const app = express()
 
-app.use(express.static(`${__dirname}/`))
+// Required for ElasticSearch proxy middleware to parse response body as JSON
+app.use(express.json())
+
+app.use('/api/search', (req, res) => {
+  if (!['GET', 'POST'].includes(req.method)) {
```

Comment:
We only allow the editor to hit the search part of ElasticSearch's API, which means either GET or POST could be used. Throw a 400 error if any other HTTP action is attempted.